docs/02-developers/04-quickstart/hardhat-ignition.md

```diff
@@ -0,0 +1,377 @@
+├── package.json       # Lists project dependencies (like a grocery list for software).
+├── hardhat.config.ts  # Configuration for Hardhat.
+├── README.md          # A file explaining your project.
+└── tsconfig.json      # Configuration for TypeScript.
+```
+### **Modules Folder**
+
+The **Modules** folder contains essential scripts used for the deployment of smart contracts. Specifically, it includes two main files: `Box.ts` and `Lock.ts`.
+
+1. #### Box.ts – Box Module
+
+This script sets up and exports a module that handles the deployment of the Box contract.
+
+```
+import { buildModule } from "@nomicfoundation/hardhat-ignition/modules";
+
+// Create and configure the BoxModule using Hardhat's Ignition library
+const BoxModule = buildModule("BoxModule", (m) => {
+  // Deploy the Box contract with no initial parameters
+  const box = m.contract("Box", []);
+
+  // Return an object containing the deployed contract
+  return { box };
+});
+
+// Export the module for use in deployment
+export default BoxModule;
+```
+
+:::info[A breakdown of each part of the Box module:]
+
+- `buildModule`: A function from `@nomicfoundation/hardhat-ignition/modules` used to create and configure modules for contract deployment.  
+- `m.contract`: Deploys a contract with the given name and constructor parameters.
+
+
+:::
+
+
+
+2. #### Lock.ts – Lock Module
+
+This script handles the deployment of the Lock contract with specific parameters for the unlock time and the locked amount.
+
+```
+import { buildModule } from "@nomicfoundation/hardhat-ignition/modules";
+
+// Define constants for unlock time and locked amount
+const JAN_1ST_2030 = 1893456000; // Unix timestamp for January 1, 2030
+const ONE_GWEI: bigint = 1_000_000_000n; // Value of 1 Gwei in Wei
+
+// Create and configure the LockModule using Hardhat's Ignition library
+const LockModule = buildModule("LockModule", (m) => {
+  // Retrieve deployment parameters with default values
+  const unlockTime = m.getParameter("unlockTime", JAN_1ST_2030);
+  const lockedAmount = m.getParameter("lockedAmount", ONE_GWEI);
+
+  // Deploy the Lock contract with the specified unlock time and initial value
+  const lock = m.contract("Lock", [unlockTime], {
+    value: lockedAmount,
+  });
+
+  // Return an object containing the deployed contract
+  return { lock };
+ });
+
+ // Export the module for use in deployment
+ export default LockModule;
+```
+
+:::info[A breakdown of each part of the Lock Module:]
+
+ - `m.getParameter`: This retrieves a deployment parameter, allowing for a default value to be specified if none is provided.  
+ 
+  **Constants**:  
+  * `JAN_1ST_2030`: The Unix timestamp for the unlock time.  
+  * `ONE_GWEI`: The value of 1 Gwei, expressed in Wei.
+
+:::
+
+
+
+  </Step>
+  <Step title="Set Up Environment Variables">
+    1. Create a file named .env in the root folder of your project.
+
+2. Add the following lines to the file:
+
+```
+RSK_MAINNET_RPC_URL=<your-mainnet-rpc-url>
```

Comment:
Without the quotes or quotes included?
RSK_MAINNET_RPC_URL="<your-mainnet-rpc-url>"